disasm.py

```diff
@@ -148,76 +148,106 @@ def addComments(script, actionLines):
                 isFloatMask = 0xF000000
                 extractMSBMask = 0xFF000000
                 extractDecimalIDMask = 0xFFFFFF
-                if parameter & isBitOrBasicMask: # if basic or bit
+                if isinstance(parameter, int) and parameter & isBitOrBasicMask: # if basic or bit
                     mostSignificantBytes = (parameter & extractMSBMask) >> (4 * 6)
                     decimalID = parameter & extractDecimalIDMask
-                    if mostSignificantBytes == int('10',16):
```

Comment:
int('10', 16) can just be written as the int literal 0x10, which is supported by python (applicable for all future instances)

Reply:
Errr nevermind I see you fixed that